requirements.txt

```diff
@@ -3,7 +3,10 @@
 
 # ptw-script\ptw-script.py: 1
 # src\check_usernames.py: 5
-jikanpy == 2.3.2
+
+# Run `pip install git+git://github.com/AWConant/jikanpy.git` for jikanpy
+# It's commented out because jikanpy isn't on PyPI
+# jikanpy == 2.3.2
```

Comment:
do we have to comment this out each time we run pigar?

Reply:
Yeah, otherwise `pip install -r requirements.txt` fails since JikanPy isn't on PyPI

Reply:
yeah i get that part i was just wondering if there's a way to let pigar know not to add it every time because this is something easy to forget

Reply:
I'm looking at the pigar github page right now, and there doesn't seem to be any way to do that unfortunately.